### PR TITLE
Feat/2.5 risk admin

### PR DESCRIPTION
**Pull request by dhadrien**

### CODE REVIEW

**Comment by dhadrien** [comment]
#96

File: contracts/protocol/lendingpool/LendingPoolConfigurator.sol [hunk 1]
```diff
@@ -220,17 +227,15 @@ contract LendingPoolConfigurator is VersionedInitializable, ILendingPoolConfigur
...
-  {
+  function updateVariableDebtToken(UpdateDebtTokenInput calldata input) external onlyPoolAdmin {
     ILendingPool cachedPool = pool;
 
     DataTypes.ReserveData memory reserveData = cachedPool.getReserveData(input.asset);
 
     (, , , uint256 decimals, ) = cachedPool.getConfiguration(input.asset).getParamsMemory();
 
-    bytes memory encodedCall = abi.encodeWithSelector(
+    bytes memory encodedCall =
```
Comment (The-3D) [inline contracts/protocol/lendingpool/LendingPoolConfigurator.sol:237]: why is this styled like this?

**Comment by The-3D** [comment]
@dhadrien please add the ability for the risk admins to call also the freeze/unfreeze reserve

File: contracts/interfaces/ILendingPoolConfigurator.sol [hunk 2]
```diff
@@ -203,14 +203,26 @@ interface ILendingPoolConfigurator {
...
+
   /**
    * @dev Initializes reserves in batch
    * @param input The array of reserves initialization parameters
    **/
   function batchInitReserve(InitReserveInput[] calldata input) external;
 
   /**
-   * @dev Updates the aToken implementation for the reserve
+   * @dev pauseReservev Updates the aToken implementation for the reserve
```
Comment (The-3D) [inline contracts/interfaces/ILendingPoolConfigurator.sol:225]: syntax error

File: contracts/protocol/lendingpool/LendingPoolConfigurator.sol [hunk 3]
```diff
@@ -464,6 +474,20 @@ contract LendingPoolConfigurator is VersionedInitializable, ILendingPoolConfigur
     _pool.setPause(val);
   }
 
+  function registerRiskAdmin(address admin) external override onlyPoolAdmin {
```
Comment (The-3D) [inline contracts/protocol/lendingpool/LendingPoolConfigurator.sol:477]: add @inheritdoc

**Comment by inkymaze** [comment]
Any update on the expect timeline for this to `master`?

**Pull request merged**